### DISCUSSION TITLE
Interchromosome translocation

### DISCUSSION THREAD

**Discussion by MehdiiYac** [GMOD/jbrowse-components#3616] [Q&A]
Hello,
I am having some trouble with generating an SV inspector view from a fasta file. I only have the fasta file an i am trying to generate a vcf or csv (or any other type of file that can be used to generate a SV inspector view). I had the following idea: 
- Running minimap2 with a file and itself (for example: minimap2 refseq.fa refseq.fa > SV_refseq.paf)
- Trying to convert the paf file to a relevant file type
But i ran into some trouble with minimap2 and i am still trying to figure out if there is a way to convert a paf file to a vcf one (or any other useful file type)
Is there an easier or just another way to generate this type of file from a fasta file?

Thanks

**Comment by cmdcolin** [comment]
Hi @MehdiiYac 
Generally, the structural variant inspector will display two types of data

1) Breakend type VCF
2) TRA calls in VCF form

you can produce this from any type of structural variant calling workflow you are interested in e.g. delly, sniffles, pbsv, or others. note that the SV inspector doesn't display plain DEL,INV,INS type events in the VCF currently, but would be good to have, but delly, sniffles, and pbsv use breakends or TRA type calls

anyways, you get your structural variant calls in VCF format, and then open it in the "import form" of the SV inspector, and it will load them into the table

hope that helps

also note: I also find it interesting to add that you can use the "synteny style" view to visualize structural variants if you have a draft genome that you are comparing to a reference, and in that case, the PAF workflow you describe may be interesting

e.g. `minimap2 reference_genome.fa draft_genome.fa > out.paf`

(can also optionally add CIGAR strings `minimap2 -a reference_genome.fa draft_genome.fa > out.paf`)

then load out.paf in the dotplot or synteny view
